docker-compose.yml

```diff
@@ -3,6 +3,8 @@ version: '3'
 services:
     tests:
         container_name: tests
+        #platform: linux/arm64
```

Comment:
Docker-compose cli supports option platform https://github.com/docker/compose/pull/8147

Reply:
That's a good point, thanks!